Marlin/example_configurations/BQ/Hephestos_2/Configuration.h

```diff
@@ -1398,6 +1367,19 @@
  */
 #define DISPLAY_CHARSET_HD44780 JAPANESE
 
+/**
+ * LCD TYPE
+ *
+ * Enable ULTRA_LCD for a 16x2, 16x4, 20x2, or 20x4 character-based LCD.
+ * Enable DOGLCD for a 128x64 (ST7565R) Full Graphical Display.
+ * (These options will be enabled automatically for most displays.)
+ *
+ * IMPORTANT: The U8glib library is required for Full Graphic Display!
+ *            https://github.com/olikraus/U8glib_Arduino
+ */
+//#define ULTRA_LCD   // Character based
+//#define DOGLCD      // Full graphics display
+
```

Comment:
This is a regression. The DOGLCD option has been removed, and the LCD options have been rearranged in the latest configs.

Reply:
@thinkyhead I'm using this as reference.
https://github.com/MarlinFirmware/Marlin/blob/3be22fc1fc712e5ccc849087290d9daf1fbc9cbf/Marlin/Configuration.h#L1387